src/main/java/reposense/parser/CsvParser.java

```diff
@@ -0,0 +1,126 @@
+package reposense.parser;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+import reposense.dataobject.Author;
+import reposense.dataobject.RepoConfiguration;
+import reposense.system.LogsManager;
+import reposense.util.Constants;
+
+/**
+ * Parses a CSV configuration file for repository information.
+ */
+public class CsvParser {
+    private static final String MESSAGE_UNABLE_TO_READ_CSV_FILE = "Unable to read the supplied CSV file.";
+    private static final String MESSAGE_MALFORMED_LINE_FORMAT = "Warning! line %d in configuration file is malformed.\n"
+            + "Contents: %s";
+
+    /** Positions of the elements of a line in the user-supplied CSV file */
+    private static final int ORGANIZATION_POSITION = 0;
+    private static final int REPOSITORY_NAME_POSITION = 1;
+    private static final int BRANCH_POSITION = 2;
+    private static final int GITHUB_ID_POSITION = 3;
+    private static final int DISPLAY_NAME_POSITION = 4;
+    private static final int ALIAS_POSITION = 5;
+
+    private static final Logger logger = LogsManager.getLogger(CsvParser.class);
+
+    /**
+     * Returns a list of {@code RepoConfiguration}, each of which corresponds to a data line in the csv file.
+     * The first line is assumed to be the header line and will be ignored.
+     * @throws IOException if user-supplied csv file does not exists or is not readable.
+     */
+    public static List<RepoConfiguration> parse(Path configFilePath) throws IOException {
+        assert (configFilePath != null);
+
+        List<RepoConfiguration> repoConfigurations = new ArrayList<RepoConfiguration>();
+        int lineNumber = 1;
+
+        try {
+            // Skip first line, which is the header row
+            final Collection<String> lines = Files.lines(configFilePath).skip(1).collect(Collectors.toList());
+
+            for (final String line : lines) {
+                processLine(repoConfigurations, line, lineNumber);
+                lineNumber++;
+            }
+
+        } catch (IOException ioe) {
+            throw new IOException(MESSAGE_UNABLE_TO_READ_CSV_FILE, ioe);
+        }
+
+        return repoConfigurations;
+    }
+
+    /**
+     * Extracts {@code Author} information from the {@code line}.
+     */
+    private static void processLine(List<RepoConfiguration> repoConfigurations, String line, int lineNumber) {
+        if (line.isEmpty()) {
+            return;
+        }
+
+        String[] elements = line.split(Constants.CSV_SPLITTER);
```

Comment:
If this is the only class that depends on this `CSV_SPLITTER` constant, initialize it in this class.
Also, use a more appropriate name maybe like `ELEMENT_SEPARATOR` or something, because CSV itself already implies comma separated.